src/model/users.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../utils/db');
+
+const User = sequelize.define('User', {
+  id: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    primaryKey: true,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  createdAt: {
+    type: DataTypes.DATE,
+    field: 'created_at',
+    allowNull: false,
+    defaultValue: DataTypes.NOW,
+  },
+}, {
+  tableName: 'users',
+  updatedAt: false,
+});
```

Comment:
```suggestion
}, {
  tableName: 'users',
});
```

Sequalize manages createdAt/updatedAt fields automatically by default. Better keep em as is.
https://sequelize.org/docs/v6/core-concepts/model-basics/#timestamps